cmd/update.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/blang/semver"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+	"github.com/spf13/cobra"
+	"os"
+)
+
+func selfUpdate() {
```

Comment:
I think `selfUpdate` function returns `error`.

Reply:
OK,  I fix it to return an error